tests/acceptance/features/bootstrap/DirectUploadContext.php

```diff
@@ -20,22 +20,24 @@ private function getLastCreatedDirectUploadToken():string {
 	}
 
 	/**
-	 * @When /^user "([^"]*)" sends a GET request to the direct\-upload endpoint with the ID of "(.*)"$/
+	 * @When /^user "([^"]*)" sends a POST request to the direct\-upload endpoint with the ID of "(.*)"$/
 	 */
 	public function userSendsAGETRequestToTheEndpointWithTheFileIdOf(
```

Comment:
need a rename, maybe something like:
```suggestion
	public function sendPOSTToDirectUploadTokenEndpointWithTheFileIdOf(
```